### PR TITLE
Version/3.21

### PR DESCRIPTION
**Pull request by lbudai**

### CODE REVIEW

File: packaging/rhel/syslog-ng.spec [hunk 1]
```diff
@@ -418,8 +418,8 @@ fi
 %{_datadir}/%{name}/tools/
 
 %changelog
-* Tue Feb 26 2019 Gabor Nagy <gabor.nagy@balabit.com> - 3.20.1-1
-- update to 3.20.1
+* Tue Feb 26 2019 Gabor Nagy <gabor.nagy@balabit.com> - 3.21.1-1
+- update to 3.21.1
```
Comment (MrAnno) [inline packaging/rhel/syslog-ng.spec]: The date does not match with the version, I'd append the changelog with a new record.

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: packaging/rhel/syslog-ng.spec (hunk 1 above)
Comment (lbudai) [reply packaging/rhel/syslog-ng.spec]: updated

Comment (MrAnno) [reply packaging/rhel/syslog-ng.spec]: Thanks

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Pull request merged**